books/kestrel/apt/isodata.lisp

```diff
@@ -1101,9 +1102,45 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(define isodata-process-undefined (undefined
+                                   (old$ symbolp)
+                                   ctx
+                                   state)
+  :returns (mv erp
+               (undefined "A @(tsee pseudo-termp) that is
+                           the translation of @('undefined').")
```

Comment:
Maybe just say `"A @(tsee pseudo-termp)."`, because if `undefined` is `:auto`, then the result is not the translation of that.